index.html

```diff
@@ -155,7 +155,7 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                 </div>
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="https://github.com/Algo-Phantoms"><i class="fa fa-github p-2 text-white" aria-hidden="true" style="background: rgba(0, 0, 0, 0.15); border-radius: 50%;"></i>&nbsp;&nbsp;
+                    <a href="https://github.com/Algo-Phantoms"><i class="fa fa-github p-2 text-white" aria-hidden="true" style="background:#3caea3; border-radius: 50%;"></i>&nbsp;&nbsp;
```

Comment:
Remove the `&nbsp;&nbsp;` which is causing unnecessary blue line on hover.

Reply:
Ok I will do that

Reply:
I have done the changes please review